eras/conway/impl/src/Cardano/Ledger/Conway/Governance/Procedures.hs

```diff
@@ -326,7 +327,7 @@ instance EraPParams era => DecCBOR (GovernanceAction era) where
       dec 2 = SumD TreasuryWithdrawals <! From
       dec 3 = SumD NoConfidence
       dec 4 = SumD NewCommittee <! From <! From
-      dec 5 = SumD NewConstitution <! From
+      dec 5 = SumD NewConstitution <! From <! From
```

Comment:
In order to use Null, you need to use `decodeNullStrictMaybe`
```suggestion
      dec 5 = SumD NewConstitution <! From <! D (decodeNullStrictMaybe decCBOR)
```

Reply:
Super thanks! :+1: